_includes/footer.html

```diff
@@ -3,7 +3,7 @@
                 <div class="small-12 medium-4 columns">
                     <div class="medium-4"><a href="{{ site.baseurl }}/index.html"><img class="footer-logo" src="{{ site.baseurl }}/img/logos/ome-main-nav.svg" alt="OME logo" /></a></div>
                     <p>&copy; 2005-{{ site.time | date: "%Y" }} University of Dundee &amp; Open Microscopy Environment. <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution 4.0 International License</a></p>
-                    <p>OME source code is available under the <a href="http://www.gnu.org/copyleft/gpl.html">GNU General public license</a> or more permissive open source licenses, or through commercial license from <a href="https://www.glencoesoftware.com" target="_blank">Glencoe Software</a>.</p>
+                    <p>OME source code is available under the <a href="http://www.gnu.org/copyleft/gpl.html">GNU General public license</a> or more permissive open source licenses, or through commercial license from <a href="https://www.glencoesoftware.com" target="_blank">Glencoe Software Incl.</a><br/>OME, Bio-Formats, OMERO and their associated logos are trademarks of <a href="https://www.glencoesoftware.com" target="_blank">Glencoe Software Incl.</a>, which holds these marks to protect them on behalf of the OME community.</p>
```

Comment:
maybe one extra `<br/>`?

Reply:
I.e. further separate the two paragraphs? I found this was at the detriment of the balance of the footer since the left column becomes much larger than the other ones.

Thoughts about moving the logo and/or the sitemap and/or the version to other columns?

Reply:
If you tried it out, don't worry about it.